github/users_administration_test.go

```diff
@@ -50,7 +50,7 @@ func TestUsersService_Suspend(t *testing.T) {
 		w.WriteHeader(http.StatusNoContent)
 	})
 
-	_, err := client.Users.Suspend(context.Background(), "u")
+	_, err := client.Users.Suspend(context.Background(), "u", "test")
```

Comment:
Please add a new test that checks that `reason` is passed successfully and keep one current test where `reason` is not passed at all.